tests/NUnit.OneTimeSetup.DreddLogs.Tests/Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Xml;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Engine;
+using NUnit.Framework;
+
+namespace NUnit.OneTimeSetup.DreddLogs.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+        private ITestRunner _testRunner;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            var testPackage = new TestPackage($"NUnit.OneTimeSetup.DreddLogs.Tests.Internal.dll");
+            _testRunner = TestEngineActivator.CreateInstance().GetRunner(testPackage);
+        }
+
+        [Test]
+        public void SimpleTest()
+        {
+            var testCaseCount = _testRunner.CountTestCases(TestFilter.Empty);
```

Comment:
I would add additional assert that testCaseCount is greater than 0 or expected some particular value (test will become more sensitive but more robust). The main reason to be sure that we will not face a situation when testCaseCount and testCaseNodes = 0 and foreach loop will not be executed.

Reply:
Yeah, makes sense